contributing/DEVELOPING.md

```diff
@@ -84,4 +84,9 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
 
 We also recommend scanning the [spec](../spec/). The non-element part should help understand some of the design aspects.
 
+## Builds and releases
+
+- The [AMP buildcop](buildcop.md) helps ensure that AMP's builds remain green (i.e. everything builds and all of the tests pass).  If you run into issues with builds that seem unrelated to your changes see if the issue is present on [Travis](https://travis-ci.org/ampproject/amphtml/branches) and let the buildcop know.
```

Comment:
Assuming we're looking to point people to the `master` builds on Travis, a better link would be https://travis-ci.org/ampproject/amphtml/builds (/builds, not /branches)

Reply:
thanks for the suggestion!  I updated the link.